### ISSUE TITLE
Can't npm install jasmine-ajax@1

### ISSUE THREAD

**Issue by artburkart** [jasmine/jasmine-ajax#121]
The README seems to suggest that jasmine-ajax for version 1.3.x can be installed, but it's not available via npm.

```bash
> npm view jasmine-ajax | grep versions
> versions: [ '2.0.1', '2.0.2', '2.99.0', '3.0.0', '3.1.0', '3.1.1' ],
```

**Comment by slackersoft** [comment]
We didn't really get npm installs working well for any of the jasmine libraries until after 2.0, so I wouldn't expect to see pre-2.0 releases there.

What wording in the [README](https://github.com/jasmine/jasmine-ajax/blob/master/README.markdown) lead you to believe 1.3 would be available on npm? I would like to change it to not be misleading.

**Comment by artburkart** [comment]
I see. That's a bummer. It was the bit that says:
> This branch is now version 2.0, if you need jasmine-ajax for Jasmine 1.3.x please grab the last release from that tag.

I guess I assumed it was available by omission, if that makes sense. I think it would be helpful if it stated somewhere that this library is only available via npm after version 2.0.0. If you disagree, I think this ticket alone is enough to prevent further confusion.

Cheers

**Comment by artburkart** [comment]
:+1:
